### PR TITLE
Insurance upload bug fix

### PR DESCRIPTION
**Pull request by michaellau-smbc**
Jira Card - File Upload issue
https://stockportbi.atlassian.net/browse/DIGITAL-8643

-) Update ValidateService to store 'isValidated' key in the cache so we know if the case reference has been validated or not to avoid multiple calls.

To Test
- Upload a file around 3mb and it should error. A small file will not work because the process time when we call aws is too short

### Checklist
- [x] Code compiles correctly
- [ ] Created tests for the new changes
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/Mappers/Element/ElementMapper.cs [hunk 1]
```diff
@@ -95,7 +95,7 @@ public async Task<object> GetAnswerValue(IElement element, FormAnswers formAnswe
             }
         }
 
-        private object GetDeclarationElementValue(string key, FormAnswers formAnswers)
+        private static object GetDeclarationElementValue(string key, FormAnswers formAnswers)
```
Comment (Teziokz) [inline src/Mappers/Element/ElementMapper.cs:98]: Check if this needs to be static

File: src/Config/appsettings.local.json [hunk 2]
```diff
@@ -16,8 +16,8 @@
   ],
   "Serilog:MinimumLevel:Override:Microsoft": "Information",
   "Serilog:MinimumLevel:Override:System": "Information",
-  "SubmissionServiceConfiguration:FakeSubmission": true,
-  "PaymentConfiguration:FakePayment": true,
+  "SubmissionServiceConfiguration:FakeSubmission": false,
```
Comment (Teziokz) [inline src/Config/appsettings.local.json:19]: Revert changes made for testing

File: src/Properties/launchSettings.json [hunk 3]
```diff
@@ -3,14 +3,16 @@
...
-      "sslPort": 44329
+      "applicationUrl": "http://localhost:60865/",
+      "sslPort": 44328
     }
   },
   "profiles": {
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
+      //"launchUrl": "email-subscription?email=michael.lau@stockport.gov.uk",
```
Comment (Teziokz) [inline src/Properties/launchSettings.json:14]: Revert changes made for testing / remove comments

**Comment by michaellau-smbc** [comment]
Wrong branch

**Pull request closed without merging**